third-party/sysdeps/common/bzip2/CMakeLists.txt

```diff
@@ -21,6 +21,8 @@ if(NOT CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
         "-DSOURCE_DIR=${_source_dir}"
       INSTALL_DESTINATION
         lib/rocm_sysdeps
+      INTERFACE_PROGRAM_DIRS
+        lib/rocm_sysdeps/bin
```

Comment:
It feels a little weird that binaries end up in `lib/rocm_sysdeps/bin` but nothing we should care about.

Reply:
Yeah, I think it's the least bad given the goal here.